marketplacetests/tests/test_marketplace_search_and_install_app.py

```diff
@@ -20,11 +20,20 @@ class TestSearchMarketplaceAndInstallApp(MarketplaceGaiaTestCase):
     # System app notification install message
     _notification_install_locator = (By.CSS_SELECTOR, '#system-banner > p')
 
+
     def test_search_and_install_app(self):
         marketplace = Marketplace(self.marionette, self.MARKETPLACE_DEV_NAME)
         marketplace.launch()
 
         self.app_name = marketplace.popular_apps[0].name
+
+        # Remove the app if already installed
+        if self.apps.is_app_installed(self.app_name):
+            self.apps.kill(marketplace.app)
+            self.apps.uninstall(self.app_name)
+            marketplace.launch()
+        marketplace.switch_to_marketplace_frame()
```

Comment:
This should also be indented inside the `if`. We do not need it to run if the `if` is false.